extensions/classifiers/LDAStringClassifier/LDAStringClassifier.py

```diff
@@ -506,6 +506,7 @@ def to_dict(self):
         model['_c_dl'] = copy.deepcopy(self._c_dl)
         model['_c_lw'] = copy.deepcopy(self._c_lw)
         model['_c_l'] = copy.deepcopy(self._c_l)
+        model = self._make_json_serializable(model)
         return model
 
     def from_dict(self, model):
```

Comment:
It's a fair question. I don't mind it being fixed in a separate PR (which might even go in before this one), but it should be fixed.